webboot/webboot.go

```diff
@@ -57,7 +65,7 @@ type Distro struct {
 
 var (
 	cmd      = flag.String("cmd", "", "Command line parameters to the second kernel")
-	ifName   = flag.String("interface", "^[we].*", "Name of the interface")
+	ifname   = flag.String("interface", "^[we].*", "Name of the interface")
```

Comment:
it's not the name of the interface, it's an RE describing one or more interfaces.

Reply:
webboot has been using that description the whole time. I can change it, I really just changed ifName to ifname and left the values the same.

Reply:
yes, my point here is you were using this re below in a function that took a name.

Change this variable name to ifRE meaning interface Regular Expression.